modules/rpmrepo/manifests/init.pp

```diff
@@ -7,6 +7,37 @@
 #
 # @example
 #   include rpmrepo
-class rpmrepo {
+class rpmrepo (
+  $repo_domain,
+  $repo_name,
+  $repo_dirs = undef,
+  $user      = 'root',
```

Comment:
Roman - it may be better to exclude $ user = 'root', $ group = 'root', $ port = ['80'], - parameters - you do not used them anywhere .. and so everywhere